sensor/common/collectorruntimeconfig/handler.go

```diff
@@ -0,0 +1,125 @@
+package collectorruntimeconfig
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/generated/internalapi/sensor"
+
+	"github.com/stackrox/rox/pkg/centralsensor"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/sensor/common"
+	"github.com/stackrox/rox/sensor/common/message"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+// Store is a store for network graph external sources.
+//
+//go:generate mockgen-wrapper
+type Store interface {
+	CollectorConfigValueStream() concurrency.ReadOnlyValueStream[*sensor.CollectorConfig]
+}
+
+// Handler forwards the collector runtime config received from Central to Collectors.
+type Handler interface {
```

Comment:
The handler is the correct approach for processing messages from Central to Sensor, but AFAIK is out of scope for this PR which focuses on sending configurations from Sensor to Collector.
I think it is better to extract this handler here to a subsequent PR.

Reply:
See my response to your other comment. I don't know how to implement this without a handler. I don't understand how splitting this into multiple PRs would work.